<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEditLampDlg</class>
 <widget class="QDialog" name="AddEditLampDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>930</width>
            <height>235</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="widget" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <widget class="QGroupBox" name="groupBox_16">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                  <horstretch>3</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="title">
                 <string>Required fields</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_46">
                 <item row="0" column="2">
                  <widget class="QToolButton" name="rbAddLampGetModels">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="text">
                    <string>r</string>
                   </property>
                   <property name="icon">
                    <iconset resource="res.qrc">
                     <normaloff>:/katynko/svg/view-refresh.svg</normaloff>:/katynko/svg/view-refresh.svg</iconset>
                   </property>
                   <property name="autoRaise">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QSpinBox" name="cbAddLampPhysicalGroup">
                   <property name="maximum">
                    <number>127</number>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="leAddLampNI">
                   <property name="maxLength">
                    <number>32</number>
                   </property>
                   <property name="clearButtonEnabled">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>Model</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_142">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text">
                    <string>NI</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="label_140">
                   <property name="text">
                    <string>Group ID</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QComboBox" name="cbxAddLampModel">
                   <property name="sizeAdjustPolicy">
                    <enum>QComboBox::AdjustToContents</enum>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0">
                  <widget class="QLabel" name="label_147">
                   <property name="text">
                    <string>Poll</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="1" colspan="2">
                  <widget class="QCheckBox" name="cbAddLampOnOffPoll">
                   <property name="text">
                    <string>Enable</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="2">
                  <widget class="QToolButton" name="tbTable">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="res.qrc">
                     <normaloff>:/katynko/svg/edit-find.svg</normaloff>:/katynko/svg/edit-find.svg</iconset>
                   </property>
                   <property name="autoRaise">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QGroupBox" name="groupBox">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="title">
                 <string>Parameters</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_3">
                 <item row="3" column="2">
                  <widget class="QSpinBox" name="sbAddLampTna">
                   <property name="correctionMode">
                    <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
                   </property>
                   <property name="minimum">
                    <number>300</number>
                   </property>
                   <property name="maximum">
                    <number>65534</number>
                   </property>
                   <property name="value">
                    <number>300</number>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="2">
                  <widget class="QSpinBox" name="sbAddLampPstart">
                   <property name="correctionMode">
                    <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
                   </property>
                   <property name="minimum">
                    <number>4</number>
                   </property>
                   <property name="maximum">
                    <number>100</number>
                   </property>
                   <property name="value">
                    <number>4</number>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="2" rowspan="2">
                  <widget class="QSpinBox" name="sbAddLampPna">
                   <property name="correctionMode">
                    <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
                   </property>
                   <property name="minimum">
                    <number>4</number>
                   </property>
                   <property name="maximum">
                    <number>100</number>
                   </property>
                   <property name="value">
                    <number>4</number>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>Pstart [%]</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="1">
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>Tna [sec]</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1" rowspan="2">
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Pna [%]</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="1" colspan="2">
                  <spacer name="horizontalSpacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QGroupBox" name="groupBox_28">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                  <horstretch>5</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="title">
                 <string>Optional</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_47">
                 <item row="2" column="0">
                  <widget class="QLabel" name="label_145">
                   <property name="text">
                    <string>Memo</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_144">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text">
                    <string>Coordinate</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="leAddLampCoordinates">
                   <property name="toolTip">
                    <string>50.43070,30.66268</string>
                   </property>
                   <property name="maxLength">
                    <number>255</number>
                   </property>
                   <property name="placeholderText">
                    <string>&lt;longitude&gt;,&lt;latitude&gt;</string>
                   </property>
                   <property name="clearButtonEnabled">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="2">
                  <widget class="QToolButton" name="tbGetPos">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="text">
                    <string>map</string>
                   </property>
                   <property name="icon">
                    <iconset resource="res.qrc">
                     <normaloff>:/katynko/svg/edit-select.svg</normaloff>:/katynko/svg/edit-select.svg</iconset>
                   </property>
                   <property name="checkable">
                    <bool>false</bool>
                   </property>
                   <property name="autoRaise">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1" colspan="2">
                  <widget class="QLineEdit" name="leAddLampMemo">
                   <property name="maxLength">
                    <number>255</number>
                   </property>
                   <property name="clearButtonEnabled">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>Street</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1" colspan="2">
                  <widget class="QLineEdit" name="leAddLampStreet">
                   <property name="clearButtonEnabled">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="1" column="0" colspan="3">
               <widget class="QLabel" name="lblShowWarn">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>4</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_62">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Add &amp;&amp; hide</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/katynko/svg/dialog-ok-apply.svg</normaloff>:/katynko/svg/dialog-ok-apply.svg</iconset>
           </property>
           <property name="shortcut">
            <string>Ctrl+Space</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbAddLamp">
           <property name="text">
            <string>Add</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/katynko/svg/draw-cross.svg</normaloff>:/katynko/svg/draw-cross.svg</iconset>
           </property>
           <property name="shortcut">
            <string>Ctrl+S</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_74">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
